Mergin/projects_manager.py

```diff
@@ -95,6 +96,10 @@ def create_project(self, project_name, project_dir, is_public, namespace):
             # User friendly error messages
             if e.http_error == 409:
                 msg = f'Project named "{project_name}" already exists in the workspace "{namespace}".\nPlease try renaming the project.'
+            elif e.http_error == 422:
```

Comment:
We can not rely only on the http error response, there could be multiple options for this code. Instead, we need to rely on our defined error codes (see https://github.com/MerginMaps/server-private/blob/fda7fc34bdbad91827391ebf4f525f4abd0c8e5b/server/src/service/errors.py#L70). It should be something like:

```suggestion
            elif e.server_code == ErrorCode.MonthlyContributorsLimitHit.value:
```